winget.ps1

```diff
@@ -26,6 +26,7 @@ winget install Yubico.YubikeyManager
 winget install GnuPG.Gpg4win
 winget install JanDeDobbeleer.OhMyPosh
 winget install Postman.Postman
+winget install Microsoft.WindowsTerminal
```

Comment:
oh, doesn't this come with default Win11 installation?

Reply:
Maybe it does. I can't upgrade to Win11 yet. Trying to figure why I'm blocked.

Reply:
thank you for the PR :)